pipelines/folioRancher/folioTestingTools/runCypressTests/Jenkinsfile

```diff
@@ -2,7 +2,8 @@
 import hudson.util.Secret
 import org.jenkinsci.plugins.workflow.libs.Library
 
-@Library('pipelines-shared-library@RANCHER-741-Jenkins-Enhancements') _
+//TODO Set to 'RANCHER-741-Jenkins-Enhancements' before merge
```

Comment:
Should it be pointed to custom branch?

Reply:
yes as it is changes to RANCHER-741-Jenkins-Enhancements